==== stdout ====

==== stderr ====
[1m[32m    Finished[0m release [optimized] target(s) in 0.34s
[1m[32m     Running[0m `target\release\examples\irradiance_volumes.exe`
[2m2024-02-24T19:15:04.708154Z[0m [32m INFO[0m [2mbevy_winit::system[0m[2m:[0m Creating new window "Bevy Irradiance Volumes Example" (0v1)
[2m2024-02-24T19:15:04.753591Z[0m [32m INFO[0m [2mbevy_render::renderer[0m[2m:[0m AdapterInfo { name: "Microsoft Basic Render Driver", vendor: 5140, device: 140, device_type: Cpu, driver: "", driver_info: "", backend: Dx12 }
[2m2024-02-24T19:15:04.811988Z[0m [33m WARN[0m [2mbevy_audio::audio_output[0m[2m:[0m No audio device found.
[2m2024-02-24T19:15:05.052586Z[0m [32m INFO[0m [2mbevy_diagnostic::system_information_diagnostics_plugin::internal[0m[2m:[0m SystemInfo { os: "Windows Server 2022 Datacenter", kernel: "20348", cpu: "AMD EPYC 7763 64-Core Processor", core_count: "2", memory: "16.0 GiB" }
[2m2024-02-24T19:15:10.585123Z[0m [31mERROR[0m [2mlog[0m[2m:[0m Device::create_render_pipeline error: Internal error in ShaderStages(VERTEX | FRAGMENT) shader: 0x887A0005    
[2m2024-02-24T19:15:10.585184Z[0m [31mERROR[0m [2mlog[0m[2m:[0m Shader translation error for stage ShaderStages(VERTEX | FRAGMENT): 0x887A0005    
[2m2024-02-24T19:15:10.585192Z[0m [31mERROR[0m [2mlog[0m[2m:[0m Please report it to https://github.com/gfx-rs/wgpu    
[2m2024-02-24T19:15:10.585251Z[0m [31mERROR[0m [2mlog[0m[2m:[0m Handling wgpu errors as fatal by default    
thread 'Async Compute Task Pool (0)' panicked at C:\Users\runneradmin\.cargo\registry\src\index.crates.io-6f17d22bba15001f\wgpu-0.19.1\src\backend\wgpu_core.rs:3009:5:
wgpu error: Validation Error

Caused by:
    In Device::create_render_pipeline
      note: label = `skybox_pipeline`
    Internal error in ShaderStages(VERTEX | FRAGMENT) shader: 0x887A0005


note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
[2m2024-02-24T19:15:10.586006Z[0m [31mERROR[0m [2mlog[0m[2m:[0m Root signature creation failed: 0x887A0005    
[2m2024-02-24T19:15:10.586077Z[0m [31mERROR[0m [2mlog[0m[2m:[0m Handling wgpu errors as fatal by default    
thread 'Async Compute Task Pool (0)' panicked at C:\Users\runneradmin\.cargo\registry\src\index.crates.io-6f17d22bba15001f\wgpu-0.19.1\src\backend\wgpu_core.rs:3009:5:
wgpu error: Validation Error

Caused by:
    In Device::create_pipeline_layout
    Parent device is lost


thread 'Async Compute Task Pool (0)' panicked at crates\bevy_render\src\render_resource\pipeline_cache.rs:702:60:
called `Result::unwrap()` on an `Err` value: PoisonError { .. }
[2m2024-02-24T19:15:10.730917Z[0m [31mERROR[0m [2mlog[0m[2m:[0m Handling wgpu errors as fatal by default    
thread 'Compute Task Pool (1)' panicked at C:\Users\runneradmin\.cargo\registry\src\index.crates.io-6f17d22bba15001f\wgpu-0.19.1\src\backend\wgpu_core.rs:3009:5:
wgpu error: Validation Error

Caused by:
    In Queue::write_buffer_with
    Creation of a resource failed for a reason other than running out of memory.


Encountered a panic in system `bevy_pbr::render::light::prepare_lights`!
thread 'main' panicked at crates\bevy_render\src\pipelined_rendering.rs:49:67:
called `Result::unwrap()` on an `Err` value: RecvError
[1m[31merror[0m[1m:[0m process didn't exit successfully: `target\release\examples\irradiance_volumes.exe` (exit code: 101)
